dropwizard-cassandra/src/main/java/smartthings/dw/cassandra/CassandraConfiguration.java

```diff
@@ -309,4 +323,106 @@ private static SSLContext getSSLContext(String truststorePath, String truststore
 		ctx.init(kmf.getKeyManagers(), tmf.getTrustManagers(), new SecureRandom());
 		return ctx;
 	}
+
+    public static class PoolConfiguration {
```

Comment:
Made my own config class here because the Cassandra Drivers PoolOptions does not have clean getter/setters to work with our yaml configuration.